weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooksEvaluationComparison.ts

```diff
@@ -548,6 +482,532 @@ const fetchEvaluationComparisonResults = async (
     evalCall => evalCall.modelRef
   );
 
+  // If we have processed imperative evaluations, we can skip the following code
+  // as it's meant for non-imperative evaluations
+  if (!isImperativeEval) {
+    populatePredictionsAndScoresNonImperative(
+      evalTraceRes,
+      result,
+      summaryData,
+      summaryOps,
+      scorerRefs,
+      predictAndScoreOps,
+      modelRefs,
+      convertISOToDate,
```

Comment:
why are you passing stuff like `convertISOToDate` here? Seems like a weird refactoring artifact